ComponentKitTests/CKComponentGestureActionsTests.mm

```diff
@@ -138,6 +138,16 @@ - (void)testThatWithNoDelegateActionsNoDelegateIsSet
   XCTAssertNil(gesture.delegate, @"Gesture delegate should not be set");
 }
 
+- (void)testThatApplyingANilRecognizerClassResultsInNoRecognizer
+{
+  CKComponentViewAttributeValue attr = CKComponentGestureAttribute(Nil, nullptr, @selector(test));
```

Comment:
@zats I agree, but the alternative is crashing/failing an assert — and I think graceful failure is better here. A legit case might look like:

```
CKComponentGestureAttribute(enabled ? [UITapGestureRecognizer class] : Nil, ...)
```

Of course, you could also just pass a nil action in that case… but meh.